### ISSUE TITLE
Craft "Item Teleport Pipe" missing

### ISSUE THREAD

**Issue by Cassetete** [tcooc/AdditionalPipesBC#146]
The craft for "Item Teleport Pipe" missing. All other crafts seem to be there except them.

Only two mods installed :
additionalpipes-4.7.2.jar
buildcraft-7.1.17.jar

Forge version 10.13.4.1614-1.7.10

**Comment by multiplemonomials** [comment]
You tried the assembly table recipe, but it's not working?

**Comment by Cassetete** [comment]
Ho, I not see the recipes for "assembly table" because no NEI for 1.8.9 (just the core, in last forge for 1.8.9, crash). I tried JEI (just enough items) but it don't show "special" recipes (like for "assembly table"). 
I searched the pattern on Internet but I found nothing other that "http://tekkitlite.wikia.com/wiki/Phased_Transport_Pipe".
The actual pattern need 8 x "diamond transport pipe", not just 2... :-(
